deployment/ansible/group_vars/staging.example

```diff
@@ -1,5 +1,7 @@
 ---
 ckan_disqus_shortname: ENTER_SHORTNAME_HERE
+#ckan_disqus_public_key: ""
```

Comment:
Might want to add a comment above this line that states something like:

"Uncomment the following lines and add your public and secret keys to enable Disqus SSO"